scripts/ci/release-npm.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+pushd $(dirname $0)
+VERSION=$(./get-version.sh)
+popd
+
+if npm view "@guardian/apps-rendering-api-models@$VERSION" > /dev/null 2>&1 ; then
+    echo "$VERSION already released. Exiting"
+    exit 0
+else
+    echo "$VERSION not yet released. Releasing"
+    # sbt "project tsApiModels" "releaseNpm $VERSION"
```

Comment:
Will this publish while this is commented out?

Reply:
😱 no! I'll uncomment 😅